### PR TITLE
Add Repo.update_all/2 example for a dependent value

### PR DESCRIPTION
**Pull request by davydog187**
Adds an example for using a value from the row when updating it.

This was the best example I could think of, open to other suggestions.

### CODE REVIEW

**Comment by greg-rychlewski** [comment]
Many of the files in this repo haven't been run through the formatter. It's best not to run the formatter on your files when submitting a PR.

**Comment by davydog187** [comment]
Thanks @greg-rychlewski, I didn't intentionally run, my editor did. Happy to submit a separate PR that formats the file first

**Comment by josevalim** [comment]
@davydog187 can you please submit it without the formatter changes? :) Eventually we will format the repo!

**Comment by davydog187** [comment]
@josevalim force pushed without the formatter changes. I'm assuming we're waiting on introducing formatter changes or should we start sending file PRs to get the process going?

File: lib/ecto/repo.ex [hunk 1]
```diff
@@ -1158,6 +1158,9 @@ defmodule Ecto.Repo do
       from(p in Post, where: p.id < 10, update: [set: [title: fragment("upper(?)", ^new_title)]])
       |> MyRepo.update_all([])
 
+      from(p in Post, where: p.id < 10, update: [set: [view_count: p.view_count * 1000]])
+      |> MyRepo.update_all([])
```
Comment (v0idpwn) [inline lib/ecto/repo.ex:1162]:
```suggestion
      from(p in Post, where: p.id < 10, update: [set: [visits: p.visits * 1000]])
      |> MyRepo.update_all([])
```
Suggesting for consistency with other examples for this function

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**